File: CPP_Programming/Day_2/program8.cpp
// Description : Write a program that accepts number from the user 
//               & print Hello that number of times

#include<iostream>
using namespace std;

void Display(int iNo)
{
    int iCnt = 0;

    for(iCnt = 1 ; iCnt <= iNo ; iCnt++)
    {
        cout<<"Hello\n";
    }

}

int main()
{
    int iValue = 0;
    
    cout<<"Enter the number : ";
    cin>>iValue;

    Display(iValue);

    return 0;
}

File: Java_Programming/Day_2/program8.java
// Description : Write a program that accepts number from the user 
//               & print Hello that number of times

import java.util.*;

class Display
{
    public void display(int iNo)
    {
        int iCnt = 0;

        for(iCnt = 1 ; iCnt <= iNo ; iCnt++)
        {
            System.out.println("Hello");
        }
    }
}

class program8
{
    public static void main(String A[])
    {
        Scanner sobj = new Scanner(System.in);

        System.out.println("Enter the number : ");
        int iValue = sobj.nextInt();

        Display dobj = new Display();

        dobj.display(iValue);

        sobj.close();
    }
}